integrations/FacebookPixel/browser.js

```diff
@@ -223,6 +244,10 @@ class FacebookPixel {
         }
       }, legacyTo);
     } else if (event === "Product Viewed") {
+      if (!isDefinedAndNotNullAndNotEmpty(prodId)) {
+        // not adding index, as only one product is supposed to be present here
+        logger.error("Product id is required. Event not sent");
```

Comment:
The event is still sent in the next statement. Either the error message or the implementation is incorrect.